### PR TITLE
Monsters added

### PR DESCRIPTION
**Pull request by LaikaFusion**
Added some monsters. And some rarities. Attention was actually paid to balance this time.

### CODE REVIEW

File: idle-rpg/game/data/monsters.js [hunk 1]
```diff
@@ -65,7 +65,128 @@ const monsters = {
       experience: 6,
       gold: 6,
       rarity: 2
-    }
+    },
+    reincareration = {
```
Comment (sizzlorox) [inline idle-rpg/game/data/monsters.js]: Wouldnt want a placeholder like this

File: idle-rpg/game/data/monsters.js [hunk 2]
```diff
@@ -65,7 +65,128 @@ const monsters = {
...
+        str: 6,
+        dex: 3,
+        end: 6
+      },
+      experience: 6,
+      gold: 6,
+      rarity: 6
+    },
+    massive = {
+      name: 'Massive',
```
Comment (sizzlorox) [inline idle-rpg/game/data/monsters.js]: Im not fond of having massive types of monsters

File: idle-rpg/game/data/monsters.js [hunk 3]
```diff
@@ -65,7 +65,128 @@ const monsters = {
...
+      stats: {
+        str: 3,
+        dex: 1,
+        end: 3
+      },
+      experience: 2,
+      gold: 3,
+      rarity: 30
+    },
+    armored = {
```
Comment (sizzlorox) [inline idle-rpg/game/data/monsters.js]: Armored rats would be weird

**Comment by isceruth** [comment]
Probably should postpone it? I mean monsters with 80 combined stats almost one shotting unlucky players would be upsetting

**Pull request merged**